### ISSUE TITLE
Table of contents crashes into main content on dataset pages.

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#725]
**Table of contents crashes into main content on dataset pages.**

On Safari at least, the headings on the left crash very distractingly into the central content when you scroll up the page.

-----
fbitem-dataset06d566fc-ac40-4283-bac4-7b4198204259
User provided contact info: @timhirsch
System: Safari 11.0.1 / Mac OS X 10.13.1
Referer: https://www.gbif.org/dataset/06d566fc-ac40-4283-bac4-7b4198204259
Window size: width 1920 - height 1032
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-12-14T15:19:20.636Z',mode:absolute,to:'2017-12-14T15:25:20.636Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-12-14T15:19:20.636Z',mode:absolute,to:'2017-12-14T15:25:20.636Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by timhirsch** [comment]
<img width="673" alt="screen shot 2017-12-14 at 10 23 32" src="https://user-images.githubusercontent.com/17124772/33999662-12801dcc-e0b9-11e7-9bcc-4893ddf261ec.png">

**Comment by MortenHofft** [comment]
Hi Tim I see that you are running System: Safari 11.0.1 / Mac OS X 10.13.1
Safari version 11 have made breaking chances that unfortunately also influence our website among many others. They have taken yet another step away from the standards. As the internet says: Safari is the new Internet explorer.

I'll see what we can do to compensate. Possible just removing TOC from browsers (like Safari 11 2017 and internet explorer 8, 2009) that don't support flex layout.

**Comment by timhirsch** [comment]
Thanks - I see it works fine on Chrome. Your option seems sensible as lack of the TOC will not be noticed much, whereas the crashing text looks like a mess.

**Comment by MortenHofft** [comment]
Yeah it does look messy. Web development is frustrating at times. Browser vendors can do an update at any time that breaks everything that worked previously. It is a wonder anything works really.

**Comment by MortenHofft** [comment]
Unable to reproduce in Saucelabs using Sierra and Safari 11. But they don't have subversions and Thomas reports from Svampe Atlas that they were only affected by a subversion of 11.

Next step is to look at the specifications of 11 vs 11.0.1 vs 11.0.2 etc. and see what have changed and compare that to the css used on the website.

**Comment by MortenHofft** [comment]
Unable to reproduce in Safari Version 10.0.1 (11602.2.14.0.7)
Unable to reproduce in Safari Version 11.0.2

Not having any browsers where i can reproduce this neither actual nor on virtual machines it is difficult to debug this one.

Perhaps this is only in Safari 11.0.1?

**Comment by MortenHofft** [comment]
@timhirsch given that this bug cannot be reproduced on other machines with same or similar versions running, then I'm hoping this issue will have resolved it self with any new version of Safari. Do you still see the issue?

**Comment by timhirsch** [comment]
I do (just checked same dataset on Safari) but if no one else is complaining then I will shut up.

**Comment by MortenHofft** [comment]
Ahh. Øv - I cannot get myself to close it then.

**Comment by MortenHofft** [comment]
From [other issue](https://github.com/gbif/portal-feedback/issues/1520#issuecomment-416941057) this crucial info has appeared:

>My version of Safari also does weird things to dataset ToCs **when zoomed out** - though no one believes me. Stick to Chrome.

Zooming out does all the difference
